src/internal/components/dataset/NamedSetList.tsx

```diff
@@ -33,6 +37,34 @@ export function NamedSetList({headerIcon, headerText, startViewId, namedSets, st
     manager.newGraph();
   };
```

Comment:
This code has changed with https://github.com/Caleydo/ordino/pull/272/files#diff-b52d524c04e27a53700c615368dd36dcb6e0aece270cd891efdb2f9c14384d64L21-L34. Please merge or re-add.